gapic/src/main/com/google/gapid/perfetto/models/Tracks.java

```diff
@@ -125,6 +125,16 @@ private Tracks() {
       }
     }
 
+    if (data.getGpu().vkEventCount() > 0) {
+      data.tracks.addLabelGroup(
```

Comment:
The group may not be necessary.  Or maybe even put this into the GPU group so that the vulkan track is closer to the gpu track.  But maybe ignore this for now and try it out in another patch.

Reply:
Done in a new commit. It is now handled in a same fashion as how GPU Queue Panel, Frame Buffer Event Panel are handled in #3719 : Only add a new group if there's more than one track.